akka-sample-distributed-workers-scala/src/main/scala/worker/Worker.scala

```diff
@@ -22,43 +15,30 @@ import scala.concurrent.duration._
 object Worker {
 
   sealed trait Message extends CborSerializable
-  case object WorkIsReady extends Message
-  case class Ack(id: String) extends Message
-  case class SubmitWork(work: Work) extends Message
+  case class DeliveredMessage(confirmTo: ActorRef[ConsumerController.Confirmed], message: WorkManager.WorkerCommand, seqNr: Long) extends Message
```

Comment:
several of those are local, so no need for them to be CborSerializable, and this would not even serializer without polymorphic annotations for the WorkerCommand

better to add the CborSerializable on the true remote messages